src/keri/app/habbing.py

```diff
@@ -2632,7 +2632,7 @@ def __init__(self, smids, mhab=None, rmids=None, **kwa):
             pre (str | None): qb64 identifier prefix of own local controller else None
             mhab (Hab | None): group member hab (local) when this Hab is multisig group
                            else None. The mhab.pre aid could be a member of
-                           .smids, or .rmids, or both.
+                           .smids, or .rmids, or both.                                                                                                                      6-0606060606
```

Comment:
What is this change?

Reply:
Looks like this was my little boy. I'll revert haha